ui/README.md

```diff
@@ -1,2 +1,37 @@
-# Anbox Cloud - Cloud Gaming Demo
+# Web UI for cloud gaming demo
 
+The Web UI for clouding gaming demo is built with [flutter](https://flutter.dev/).
+
+## Requirements
+
+To setup Web development and build the UI, make sure that you have flutter installed.
+Please follow the [instructions](https://docs.flutter.dev/get-started/install/linux) to install the flutter on your machine.
+
+## Build
+
+To build the UI, simply run:
+
+    ```
+    flutter build web
+    ```
+
+This generates assets, js, html files that are required to run the Web UI and places all of them into the `build/web` folder.
+Please refer to the official [documentation](https://docs.flutter.dev/get-started/web) for more details on how to build Web apps with flutter.
+
+## Deploy
+
+Create a `static` folder under the backend service and copy everything that is in the `build/web` folder to the `static` folder.
+
+    ```
+    mkdir -p backend/service/static
+    cp av build/web/* backend/service/static
+    ```
+
+This enables Web UI to be hooked up with the backend service.
+Then launch the backend service with the script:
```

Comment:
```suggestion
Then launch the backend service with the following script:
```

Reply:
No, any code change made to ui requires people to rebuild the UI manually by running `flutter build web` command and putting the `build/web` folder into place `backend/service/static/`. That's not installation script is intended to do.

Reply:
What if you don't do any code changes? Then you don't need to copy it, or do you?

Reply:
No matter if we made a code change to UI or not, we should always ensure the build artifacts(ui/build/web/*) exist under `backend/service/static/` so that the backend service can load it on startup.

Reply:
But that folder does not exist in the repo. So if you just run the demo without changes, the folder is either created automatically by the installation script or we're missing a step in the installation process.

Reply:
No, as I mentioned above, the install script won't create that folder since we don't run `flutter build web` in that script to build web UI. 
See https://github.com/anbox-cloud/cloud-gaming-demo/blob/main/snap/snapcraft.yaml#L56
The folder was created when building the snap package. See here
https://github.com/anbox-cloud/cloud-gaming-demo/blob/main/snap/snapcraft.yaml#L56
and we copy that build artifacts to $SNAPCRAFT_PART_INSTALL/service/static where the backend service loads web UI from. 
That's being said, the snap package contains web UI + backend to get the demo running next to the Anbox Cloud appliance.
Generally, for the people who are just interested in installing the cloud gaming demo,  they run the install scripts which will help them to
1. install the snap (webUI  + backend)
2. install the games (APKs) to AMS
3. configure the route and backend service
After everything setup well, they will have demo service up and running next to the appliance. 

The DEVELOP.MD file that we cook up here exists for the people who want to modify the code or build something on top of this demo. it would guide them on 
1. how to set up the development environment
2. build the web UI
3. configure the backend service
And this facilitates the development and they can hand on it easily and quickly without building the snap and deploying everything from scratch whenever they make a code change to web UI/backend.

Reply:
OK, but then "The backend service will then use this version of the web UI." should work, right?

Reply:
yes, that's right.`.